bootstrap/data_ingester.py

```diff
@@ -25,53 +25,49 @@
 CONTENT_TYPE = "application/json"
 
 
-def get_dataset_id(create_dataset_url, headers, dataset_title, schema_id):
+def get_dataset_id(create_dataset_url, headers, dataset_title, schema_id, data):
     """
     Get the datasetId by making a POST Request to "/data/foundation/catalog/datasets?requestDataSource=true"
     :param create_dataset_url: url
     :param headers: headers
     :param dataset_title: dataset title
     :param schema_id: schema url
+    :param data: post request data
     :return: dataset id
     """
-    data_for_create_dataset = {
-        "schemaRef": {
-            "id": schema_id,
-            "contentType": "application/vnd.adobe.xed+json; version=1"
-        },
-        "name": dataset_title,
-        "description": dataset_title,
-        "fileDescription": {
-            "persisted": True,
-            "containerFormat": "parquet",
-            "format": "json"
-        },
-        "aspect": "production"
-    }
+
+    # Set the title and description
+    data['name'] = dataset_title
+    data['description'] = dataset_title
```

Comment:
Should this be, `dataset_description`?

Reply:
Does not matter for all the inputs and outputs class, schema and mixins using the title as the description to optimize the number of key, value pairs in the config file